mne/coreg.py

```diff
@@ -168,6 +179,38 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
             shutil.copy(mne_fname % name, dest_bem)
 
 
+def create_high_res_head(subject, subjects_dir, fs_home):
+    """Wrapper for FreeSurfer $ mkheadsurf
+
+    Parameters
+    ----------
+    subject : str
+        Subject.
+    subjects_dir : str
+        Subjects directory.
+    fs_home : str
+        Path to the FreeSurfer installation.
+    """
+    if not os.path.exists(fs_home):
+        raise ValueError("Invalid fs_home parameter: %s does not exist" %
+                         fs_home)
+
+    env = os.environ.copy()
+    env['SUBJECTS_DIR'] = subjects_dir
+    env['SUBJECT'] = subject
+    env['FREESURFER_HOME'] = fs_home
+    env['PATH'] = os.path.join(fs_home, 'bin') + ':' + env['PATH']
+
+    subj_path = os.path.join(subjects_dir, subject)
+    if os.path.exists(os.path.join(subj_path, 'mri', 'T1.mgz')):
+        mri = 'T1.mgz'
+    else:
+        mri = 'T1'
+    run_subprocess('source $FREESURFER_HOME/FreeSurferEnv.sh; ' +
+                   'mkheadsurf -subjid ' + subject + ' -srcvol ' + mri,
+                   env=env, shell=True)
```

Comment:
I don't like this much as it means computation is done by the GUI while we should document how it should be done with mne command. If you don't agree with how the mne command does stuff the fix should be there.